src/Symfony/Bridge/Doctrine/Form/ChoiceList/IdReader.php

```diff
@@ -91,7 +91,7 @@ public function isIntId(): bool
     public function getIdValue($object)
     {
         if (!$object) {
-            return null;
+            return '';
```

Comment:
This looks wrong to me. the id of `null` is not the empty string.

Reply:
`IdReader::getIdValue` is used as `choice_value` callback, it must always return a string actually.